resources/authgear/templates/en/web/settings.html

```diff
@@ -142,12 +144,14 @@ <h2 class="grid-area-title margin-0 primary-txt text-base font-semibold">
   {{ end }}
   {{ end }}
 
+  {{ if $show_more_button }}
   <section class="padding-20 row-sep">
     <a class="link action text-sm" href="/settings/identity">
       <i class="ti ti-chevrons-right"></i>
       {{ template "more-button-label" }}
```

Comment:
The label has to be updated. See https://github.com/authgear/authgear-server/issues/1061#issuecomment-890772930

Reply:
Ping

Reply:
Ok, I will update it